src/compas/geometry/_primitives/plane.py

```diff
@@ -243,8 +243,8 @@ def transform(self, matrix):
             The transformation matrix.
 
         """
-        point = transform_points([self.point], matrix)
-        normal = transform_vectors([self.normal], matrix)
+        point = transform_points([self.point], matrix)[0]
```

Comment:
Was this a bug? Does it fail without this change? Maybe you could add a little unit test for it (especially if it was failing before and your change fixes it)..

Reply:
sorry, I just see the linked issue ;)